test/e2e/tests/editor/events.spec.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require('@wordpress/e2e-test-utils-playwright');
+
+test.describe('Events in the Editor', () => {
+	test.beforeEach(async ({ admin, page }) => {
+		await admin.createNewPost({ postType: 'gatherpress_event' });
+		await page.getByLabel('Add title').fill('Change title to allow saving');
+		await page
+			.getByLabel('Block: Event Date')
+			.locator('div')
+			.first()
+			.isVisible();
+		await page.getByRole('heading', { name: 'Date & time' }).isVisible();
```

Comment:
This will have no impact, as the `locator.isVisible()` utility only checks whether an element is visible, returning a boolean value. To assert visibility, the `await expect(locator).toBeVisible()` assertion needs to be used.